cpp/lib/ome/test/main.cpp

```diff
@@ -45,3 +45,11 @@ main (int   argc,
   testing::InitGoogleTest(&argc, argv);
   return RUN_ALL_TESTS();
 }
+
+bool
+verbose ()
+{
+  static bool v(getenv("BIOFORMATS_TEST_VERBOSE")
+                && std::string(getenv("BIOFORMATS_TEST_VERBOSE")) == "true");
```

Comment:
I guess in part it depends on if you already have some convenient function handy analogous to Java's `Boolean.parseBoolean(String)`; suppose you could at least flatten the case.